basics/installation/environments/docker.md

```diff
@@ -339,7 +339,7 @@ services:
       image: phpmyadmin/phpmyadmin
       container_name: phpmyadmin
       links:
-        - some-mysql
+        - mysql
       environment:
         PMA_HOST: some-mysql
```

Comment:
```suggestion
        PMA_HOST: mysql
```
This also should be changed to `mysql` as the host must match the database service name too